### ISSUE TITLE
MICKA forces vertical reference system and extent when not appropriate

### ISSUE THREAD

**Issue by nmtoken** [GeoEra-GIP/Project-Support-WP8#438]
Comment from HOVER metadata creator for a 2D TIFF dataset

> MICKA seems to be forcing me to put the file "Presentation Form" as "Model digital".  This would be fine (and accurate), but in doing this it's forcing me to define a vertical reference system and extent, which isn’t appropriate for this data.  I have put in some dummy values to these fields to make the metadata record valid, but they are basically meaningless.

Is there some assumption in MICKA that model implies geographic x/y and height/depth?

Is it possible to set vertical reference to

    <gmd:verticalElement gco:nilReason="inapplicable">

**Comment by Kramolisova** [comment]
In progress across the CGS metadata team, we have a discussion on how to proceed in solving the problem correctly.

**Comment by Kramolisova** [comment]
I am not sure which data you would like to describe. But we should emphasize that the Digital model is only given to describe 3D (or multi-D) models. May be for you it is better to fill in item 12 Presentation Form with value "Image Digital", it is usually described in the standard as an image from a sensor.  Or possibly "Map Digital", which is explained as "map represented in raster or vector form". So it depends on what you want to describe.

**Comment by nmtoken** [comment]
https://egdi.geology.cz/record/basic/600ee753-5b98-4805-af06-23100a010833

**Comment by nmtoken** [comment]
> But we should emphasize that the Digital model is only given to describe 3D (or multi-D) models.

In the code list, 3D is not mentioned only `multi-dimensional`.  In my dictionary, _multi-_ means more than one, so 2D would qualify here, and is the best description.  This is not captured sensor data.  

**ref:** https://standards.iso.org/iso/19139/resources/gmxCodelists.xml#CI_PresentationFormCode


    <CodeDefinition gml:id="CI_PresentationFormCode_modelDigital">
      <gml:description>multi-dimensional digital representation of a feature, process, etc.</gml:description>
      <gml:identifier codeSpace="ISOTC211/19115">modelDigital</gml:identifier>
    </CodeDefinition>

_mapDigital_ could work but it seems less correct semantically

    <CodeDefinition gml:id="CI_PresentationFormCode_mapDigital">
      <gml:description>map represented in raster or vector form</gml:description>
      <gml:identifier codeSpace="ISOTC211/19115">mapDigital</gml:identifier>
    </CodeDefinition>

**Comment by nmtoken** [comment]
So just to understand the process, the person creating the metadata has _I assume_ chosen Digital model, from the given list of presentation forms and this (alone) is what then leads to a request for a vertical CRS?  

This still seems problematic, what if the model dimensions don't include height/depth, but rather some other dimensions like time, pressure...

**Comment by Kramolisova** [comment]
You are right user when chosen Digital model, from the given list of presentation forms  then leads to a request for a vertical CRS, user then have 3 more Vertical items to fill to have metadata record valid, but user can let this items empty if information missing or do not make sense.
And we can discuss amendment of profile and extension of metadata profile form for some other dimensions like time, pressure...if there is needs from projects. IS there any item in metadata ISO? We could look for them.

**Comment by nmtoken** [comment]
For Time as a dimension (as an extent) we have both _gmd:EX_TemporalExtent_ and _gmd:EX_SpatialTemporalExtent_ but I can't see how you might add other types of dimensions.

For other dimensions as part of `<gmd:referenceSystemInfo><gmd:MD_ReferenceSystem><gmd:referenceSystemIdentifier>` then you can certainly reference either individual dimensions like (from the INSPIRE default CRS listing) 'http://codes.wmo.int/grib2/codeflag/4.2/_0-3-3`  for Pressure coordinate in the free atmosphere (ICAO international standard atmosphere).  Or compound CRS including this and other dimensions

**Comment by Kramolisova** [comment]
There is still a temporal extent in the metadata, which we also use, there are no other dimensions in metadata. (And yes, it's still gmd: EX_SpatialTemporalExtent), but we find it unnecessary to use it for metadata.

Yes, we could add some information in the referenceSystemInfo, if it will be required and a code list will be suggested in the future.

**Comment by nmtoken** [comment]
So the metadata record has been now updated to remove the vertical CRS information, but the record is now showing as invalid

**Comment by Kramolisova** [comment]
When Metadata element **12 Presentation Form** is filled vith value **Model Digital** Validation is changed and requires vertical elements. Value **Model Digital** indicates that this is metadata record for 3D model so Validation is changed. It is described on page 20 of the Cookbook (https://egdi.geology.cz/layout/egdi/MetadataCookbook_Lite.pdf ). May be we should describe it in more details.
But these vertical items are not required by INSPIRE, just for EGDI Metadata 3D Profile.

If you datset is not for 3D model, you sould delete **Model Digital** and then validation changed again and vertical items will be not required.
 
![image](https://user-images.githubusercontent.com/46894474/118950617-a583b700-b95a-11eb-9d9a-c52f456a2539.png)

**Comment by Kramolisova** [comment]
Thank you, James, we will have to discuss this issue in more detail within the CGS metadata team.

**Comment by Renadani1997** [comment]
Life is not kaney

**Comment by Renadani1997** [comment]
Klaipeda
